PocketKit/Sources/PocketKit/MyList/SearchItemsList/SkeletonView.swift

```diff
@@ -0,0 +1,57 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+
+struct SkeletonListView: View {
+    var body: some View {
+        List {
+            ForEach(1..<10) { _ in
+                HStack {
+                    ListItem(model: SearchItem(item: DummyData()))
+                    Spacer()
+                }
+            }
+        }
+        .listStyle(.plain)
+        .accessibilityIdentifier("skeleton-view")
+        .redacted(reason: .placeholder)
+    }
+}
```

Comment:
this is the `redacted` method, should I remove this or leave this here for now? We're not using this to maintain consistency with our other skeleton cells, but when we move to SwiftUI. I personally think it would be useful to use this as it is a better representation of the data.

Reply:
as discussed live with @timc-mozilla and @Gio2018, I will go ahead and removed this unused code from the codebase.